packages/client/src/generated/ethereum/api.ts

```diff
@@ -312,15 +318,15 @@ export const V1ApiAxiosParamCreator = function (configuration?: Configuration) {
             };
```

Comment:
do we have an endpoint to get network fees?

Reply:
yes, we have `/gas/price` endpoint: https://github.com/shapeshift/unchained/blob/244f05ff15bfd0ebf35a14a83b702a36cf9c7d88/coinstacks/ethereum/api/src/controller.ts#L192